Source/Core/DolphinQt/Config/Mapping/IOWindow.cpp

```diff
@@ -532,7 +562,29 @@ void IOWindow::UpdateDeviceList()
       QString::fromStdString(m_controller->GetDefaultDevice().ToString()));
 }
 
-InputStateDelegate::InputStateDelegate(IOWindow* parent) : QItemDelegate(parent), m_parent(parent)
+void IOWindow::UpdateExpression(std::string new_expression)
+{
+  const auto lock = m_controller->GetStateLock();
+  if (new_expression != m_reference->GetExpression())
```

Comment:
Would this be a candidate for invert-condition/early-exit, since the remainder of the method is just inside this `if`?

Reply:
Yup.